### PR TITLE
added a new helmet in the form of a cthulhu mask.

### PR DESCRIPTION
**Pull request by brainfood1183**
Modified entities/clothing/head/misc.yml

added cthulhu.rsi file to textures/clothing/head/misc

## About the PR 

**Screenshots**

**Changelog**

:cl:
- add: Added fun!
- remove: Removed fun!

### CODE REVIEW

File: Resources/Prototypes/Entities/Clothing/Head/misc.yml [hunk 1]
```diff
@@ -153,3 +153,15 @@
     sprite: Clothing/Head/Hats/catears.rsi
   - type: AddAccentClothing
     accent: OwOAccent
+
+- type: entity
+  parent: ClothingHeadBase
+  id: ClothingHeadHatCthulhu
+  name: Cthulhu Mask
```
Comment (Macoron) [inline Resources/Prototypes/Entities/Clothing/Head/misc.yml:160]: Trademark and IP for Cthulhu is... Difficult...
https://lovecraft.fandom.com/wiki/Copyright_status_of_works_by_H._P._Lovecraft

Maybe rename it something neutral, like "Elder God Mask" or "Squid Mask"?

File: Resources/Textures/Clothing/Head/Misc/cthulhu.rsi/meta.json [hunk 2]
```diff
@@ -0,0 +1,26 @@
+{
+  "version": 1,
+  "license": "CC-BY-SA-3.0",
+  "copyright": "Taken from https://github.com/brainfood1183/space-station-14",
```
Comment (Macoron) [inline Resources/Textures/Clothing/Head/Misc/cthulhu.rsi/meta.json:4]: You need to show exact commit like this:
```
Taken from tgstation at commit https://github.com/tgstation/tgstation/commit/547852588166c8e091b441e4e67169e156bb09c1
```

But I would suggest you to add more simpler copyright that used for other original works. For example:
```
"Veritius#2351 (257233913951289344) on discord / @Veritius on GitHub"
```

**Review by Macoron — COMMENTED** [review]
Please make changelog without "add: Added fun..."

File: Resources/Prototypes/Entities/Clothing/Head/misc.yml (hunk 1 above)
Comment (brainfood1183) [reply Resources/Prototypes/Entities/Clothing/Head/misc.yml:160]: basically cthulhu himself is in the public domain as he was a creature created prior to the trademarking, the novel call of cthulhu however is not subject to public domain but the character is free domain. 

Cthulhu himself is very much in the public domain, as is the majority of Lovecraft's fiction. It's not copyrights that are the hazard here, it's trademarks. "Call of Cthulhu" is an established trademark in gaming, variously owned and licensed. Ditto for other game titles you might already know

File: Resources/Textures/Clothing/Head/Misc/cthulhu.rsi/meta.json (hunk 2 above)
Comment (brainfood1183) [reply Resources/Textures/Clothing/Head/Misc/cthulhu.rsi/meta.json:4]: thanks, i wasnt really sure what to put for copyright.

File: Resources/Prototypes/Entities/Clothing/Head/misc.yml (hunk 1 above)
Comment (brainfood1183) [reply Resources/Prototypes/Entities/Clothing/Head/misc.yml]: i changed the name of the item and description to a more generic Eldritch. I cannot however figure out how to change the .rsi file name from cthulhu to eldritch although it probably doesnt matter as much?

Comment (Macoron) [reply Resources/Prototypes/Entities/Clothing/Head/misc.yml]: > how to change the .rsi file name from cthulhu to eldritch

You just rename folder and path to it. Thats it.

**Review by EmoGarbage404 — CHANGES_REQUESTED** [review]
as it stands, this is unused content. Consider adding it to either the autodrobe or even the emagged pietyvend.

**Pull request closed without merging**